data/parameters/array-simulator_parameters.yml

```diff
@@ -0,0 +1,25 @@
+simtelDataDirectory:
+  len: 1
+  unit: null
+  default: null
+primary:
+  len: 1
+  unit: null
+zenithAngle: 
+  len: 1
+  unit: !astropy.units.Unit {unit: deg}
```

Comment:
Are these yaml files supposed to be produced by the user? If so, can't the user just write "deg" in the unit and we complete it to full astropy syntax in the code later? It will be much more user friendly that way.

I am not sure if this was already discussed before, haven't really been keeping up, so sorry if it has.

Reply:
No, these are pre-defined parameter file. The user is not supposed to touch it.
This way of writing the units is how the yaml module from astropy can read it.

Reply:
OK!